feature/vmaf.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What are these?

Reply:
this is something that slipped through, some random files that sometimes get generated, will remove